tests/test_c_parser.py

```diff
@@ -88,12 +88,15 @@ def parse(self, txt, filename=''):
     def setUp(self):
         self.cparser = _c_parser
 
-    def assert_coord(self, node, line, file=None):
+    def assert_coord(self, node, line, column, file=None):
         self.assertEqual(node.coord.line, line)
+        if column:
```

Comment:
In fact, the `column` implemented keeps the semantic of lex, so the column always starts at 1, not 0.
Do you want me to test against `None` (plus an assert against 0)?

By the way, do you agree to keep the semantic of lex (starting column AND lineno at 1) or do you want a modification to starts column at 0?